app/main/page.tsx

```diff
@@ -54,12 +56,50 @@ function App() {
     foregroundDrawer.drawHandWave(handWave);
   };
 
-  //TODO: store the new nametag content into DB
   const updateNameTagContent: SubmitHandler<NameTagContent> = (data) => {
     setNameTagContent(data);
     foregroundDrawer.drawNameTag(data);
+
+    // Update nametag in DB
+    updateNameTagInDB(data as NameTagContent);
+  };
+
+  const fetchNametagFromDB = async () => {
+    const session = await getSession();
+
+    if (session && session.user) {
+      await fetch("/api/auth/users/fetchUserData/nameTag", { 
+        method: "POST",
```

Comment:
Shouldn't we use a GET request to query the DB and POST to make updates?

Reply:
Good idea! Modified it to use GET instead of post (https://github.com/aimpowered/ZoomCompanion/pull/60/commits/983af0397d7ba9a3eb19235fbca295012120bede#diff-289ecc025262610eb95cd023e7ab6b2bd5f6ca72ef074ee564ae159ce8706fe5)